### DISCUSSION TITLE
Are there other models in your roadmap?

### DISCUSSION THREAD

**Discussion by karl6211** [logancyang/obsidian-copilot#29] [General]
Hi Logan,

Just wanted to thank you for your plugin!. I've been looking for a long time for the perfect AI integration with my Obsidian notes, but as a non-coder, I've been really frustrated with all the advice on training AI chatbots on my own notes. 

I really like how Copilot can read a note as context and I can chat about it. But I was wondering if your plugin could have a choice of models to play with like with Quora's Poe app?  So that we can keep the costs down, especially when my notes generate so many tokens at OpenAi.

Many thanks again, and keep up the good work.

**Comment by logancyang** [comment]
Hi @karl6211 , thanks for being an early adopter of this plugin! I'm actually actively looking into adding local models that can work without an internet connection. It will be great for privacy and cost reduction. Please stay tuned!

**Reply by karl6211** [reply]
That's awesome, Logan!. I'll definitely stay tuned. Thanks!

**Comment by yilu1015** [comment]
Can I make a recommendation for Claude 2? It can handle far larger tokens than GPT -- think novel-sized blocks of text -- and is perfect for people with large vaults like me. Thanks for all your work! Can't wait for future developments of this plugin.

https://claude.ai

**Reply by logancyang** [reply]
Hi @yilu1015 , thanks for the suggestion. Do you have Claude API access? I've been waiting forever, they still haven't given me the access.

**Reply by writtenfool** [reply]
I do. Claude AI provides much better information for me. Looking forward to using it directly in Obsidian.

**Reply by writtenfool** [reply]
If you need a Claude tester feel free to ask. I'm currently using my API key with Visual Studio via extension CodeGPT and the standalone Windows app Chatbox. Both work well, but having in Obsidian would be very nice.

**Reply by logancyang** [reply]
Thanks @writtenfool ! I actually had a PR to enable Claude, will get back to it soon. OpenAI hasn't been stable recently, Claude is going to be a great addition to the plugin.

**Reply by writtenfool** [reply]
No worries. Many thank yous!

**Reply by writtenfool** [reply]
Thank you @logancyang. Co-pilot works flawlessly for me in Obsidian. My PKM and LLM world is complete. Cheers!

**Reply by logancyang** [reply]
@writtenfool Thanks for the support!!
